modules/recommended-configurable-storage-technology.adoc

```diff
@@ -23,14 +23,15 @@ technologies for the given {product-title} cluster application.
 .Recommended and configurable storage technology
 [options="header"]
 |===
-|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics|Logging|Apps
+|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics footnoteref:[metrics-prometheus,Prometheus is the the underlying technology used for metrics.]|Logging|Apps
 
 | Block
 | Yes footnoteref:[disk,This does not apply to physical disk, VM physical disk, VMDK, loopback over NFS, AWS EBS, and Azure Disk.]
 | No
 | Configurable
 | Not configurable
-| Recommended
+| Recommended footnoteref:[metrics-warning,For metrics, it is an anti-pattern to
```

Comment:
@jboxman I am not familiar with the term `any-pattern`. Is this the best word to use here? I don't see it used elsewhere in the docs.

Reply:
It's referring to this I believe: https://en.wikipedia.org/wiki/Anti-pattern